Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -109,6 +109,10 @@
     netsync: false
     sprite: Objects/Specific/Hydroponics/aloe.rsi
     state: cream
+  - type: Stack
+    stackType: AloeCream
```

Comment:
somehow missed this, this stack type doesn't actually exist and needs to get added in `Resources/Prototypes/Stacks/medical_stacks.yml`

Reply:
alright fixed it 😅

Reply:
hopefully